acceptance_tests/features/environment.py

```diff
@@ -1,5 +1,9 @@
 from acceptance_tests import browser
-from controllers import database_controller
+
+from controllers import collection_exercise_controller, database_controller, sample_controller
+from controllers import party_controller, django_oauth_controller, case_controller
+from config import Config
+from acceptance_tests.features.steps import common
```

Comment:
The order of these imports is a bit wonky, should be alphabetised and grouped together